cmd/glasskube/cmd/update.go

```diff
@@ -60,6 +60,9 @@ func printTransaction(tx update.UpdateTransaction) {
 		if item.UpdateRequired() {
 			fmt.Fprintf(w, "%v:\t%v\t-> %v\n",
 				item.Package.Name, item.Package.Spec.PackageInfo.Version, item.Version)
+			for _, req := range tx.Requirements {
+				fmt.Fprintf(w, "%v:\t-\t-> %v\n", req.Name, req.Version)
+			}
```

Comment:
I think this should be done after the for loop over tx.Items, since the dependencies are collected on the transaction, not per item. If there are multiple items, the dependencies would be listed for each item.

Reply:
I had one job